GSoC/2016/README.md

```diff
@@ -1,32 +1,33 @@
 
-However, the project will go on, if anyone have interests in this idea, please check [QuCmp.jl](https://github.com/JuliaQuantum/QuCmp) in JuliaQuantum's repo.
+In any case, the project has to move on. For your interest, please check [QuCmp.jl](https://github.com/JuliaQuantum/QuCmp) in JuliaQuantum's repo.
 
-To prevent the same accident from happening again and to help people in region without Google services to get involved with all those amazing ideas, I'll leave some advise here for other students from where google is not available :
+To prevent the same tragedy happening on future applicants, especially to help applicants in regions without Google services, I am leaving some advice below.
 
 ### For students
 
-- If you have trouble in connecting GSoC's official website, (eg. the browser gives you 404 error), please ask JuliaQuantum members for help by email or gitter.
-- check your application status after your application regularly
+- If you have trouble in connecting GSoC's official website, eg. the browser gives you 404 error, you should ask JuliaQuantum members for help by email or gitter. Make sure you have read the rules and regulations before applying, with all the deadlines in mind.
+- Check your application status regularly after your application.
 - if you do not use gmail very often or you have trouble in using gmail, please set your gmail automatically forward the emails to your commonly used email
-- Besides your proposal, check the enrollment letter, it has to be your university's official certificate in English, which could take some time in the preparation.
+- If you do not use gmail very often or you can hardly use gmail, please set your gmail to automatically forward emails to your commonly used email address. The organizers from Google will send emails to your Gmail address by default.
+- Prepare your enrollment letter early before the deadline, which could take some time and has to be your university's official certificate in English.
 
 ### For mentors
 
-Since JuliaQuantum mentors may not originally come from its present members, and it's possible to invite people as your mentor, some mentors from China could be the first time in participating. So here are some suggestions for mentors from places that may have trouble in using Google services.
+By courtesy of JuliaQuantum's policy that non-organizational person can join the organization by passing a qualification review process and act as a mentor of the applicant, it is possible for outsiders to become a GSoC mentor under the organization's umbrella. Again, if there is any communication problems with Google or understanding the rules, the organization could offer a hand.
 
 - You may have trouble in reading through the mentor's guide and rules in GSoC's official website. If so, please feel free to ask in JuliaQuantum's email list or gitter channel for help.
```

Comment:
~~- You may have trouble in reading through the mentor's guide and rules in GSoC's official website. If so, please feel free to ask in JuliaQuantum's email list or gitter channel for help.~~